bokeh/io.py

```diff
@@ -377,15 +375,12 @@ def save(obj, filename=None, resources=None, title=None, state=None, validate=Tr
     Returns:
         filename (str) : the filename where the HTML file is saved.
 
-    Raises:
-        RuntimeError
-
     '''
     if state is None:
         state = _state
 
     filename, resources, title = _get_save_args(state, filename, resources, title)
-    _save_helper(obj, filename, resources, title, validate)
+    _save_helper(obj, filename, resources, title)
```

Comment:
need to warn that the `validate` arg has been no-op'ed. Now it always happens inside of `_ModelInDocument`